Plugins/BuildServerIntegration/AzureDevOpsIntegration/AzureDevOpsAdapter.cs

```diff
@@ -11,6 +14,8 @@
 using GitUIPluginInterfaces.BuildServerIntegration;
 using Microsoft.VisualStudio.Threading;
 
+[assembly:InternalsVisibleTo("AzureDevOpsIntegration.Tests")]
```

Comment:
Attributes like this are better kept in dedicated files like AssemblyInfo.cs or InternalsVisibleTo.cs.